property-groups/background/README.md

```diff
@@ -0,0 +1,29 @@
+description
+
+Example JSON:
+```
+{
+    "background":{
+    "image":{
+        "position":"center center",
+        "repeat":"no-repeat",
+        "src":"https://examples/example.png"
```

Comment:
I would put the src to the top, because that seems to be the most important :)

Reply:
fix it on both the JSON and the table